external-crates/move/crates/move-compiler/src/parser/ast.rs

```diff
@@ -1849,12 +1852,19 @@ impl AstDebug for PathEntry {
 
 impl AstDebug for NamePath {
     fn ast_debug(&self, w: &mut AstWriter) {
-        let NamePath { root, entries } = self;
+        let NamePath {
+            root,
+            entries,
+            is_incomplete,
+        } = self;
         w.write(format!("{}::", root));
         w.list(entries, "::", |w, e| {
             e.ast_debug(w);
             false
         });
+        if *is_incomplete {
+            w.write("_");
```

Comment:
```suggestion
            w.write("::_");
```

Reply:
It's actually even a bit trickier than that on the second thought. We should have the following to handle both case when `entries` is empty and when it's not:
```
        if *is_incomplete {
            if !entries.is_empty() {
                w.write("::");
            }
            w.write("_#incomplete#_");
        }
```